schemas/settings.gschema.xml

```diff
@@ -98,6 +98,11 @@
             <summary>Focus effect</summary>
             <description>Define which focus effect to apply when using tiling layouts</description>
         </key>
+        <key name="workspace-appmenu" type="b">
```

Comment:
Can we rename the property to `show-application-launcher`?

Reply:
Property name has been changed